packages/docs/.vuepress/config.js

```diff
@@ -12,6 +12,24 @@ module.exports = {
         'og:image': '/images/og-logo.png',
       },
     ],
+    [
+      'link', {
+        'rel': 'preconnect',
+        'href': 'https://fonts.gstatic.com',
+      },
+    ],
+    [
+      'link', {
+        'rel': 'stylesheet',
+        'href': 'https://fonts.googleapis.com/css2?family=Public+Sans:ital,wght@0,400;0,600;1,400;1,600&display=swap',
+      },
+    ],
+    [
+      'link', {
+        'rel': 'stylesheet',
+        'href': 'https://fonts.googleapis.com/css2?family=Inconsolata&display=swap',
```

Comment:
This was missed before? We didn't have the monospace font family included?

Reply:
Correct; storybook and prism were pulling it in, but it was never included in the docs build. If I remember correctly, we had been waiting on official sign-off from Brand. Not sure what happened there, but the time for waiting is over!